packages/release-please/test/release-please.ts

```diff
@@ -696,7 +708,7 @@ describe('ReleasePleaseBot', () => {
 
         sinon.assert.calledOnce(createPullRequestsStub);
         sinon.assert.calledOnce(createReleasesStub);
-        sinon.assert.calledOnce(fromManifestStub);
+        sinon.assert.calledTwice(fromManifestStub);
```

Comment:
@chingor13 This is now called twice. Can you help me understand why?

Reply:
1st call:

<img width="1442" alt="Screenshot 2025-03-02 at 5 46 32 PM" src="https://github.com/user-attachments/assets/fb65cfc2-0f98-44a0-93be-f8e812a3cbd7" />

2nd call:

<img width="1441" alt="Screenshot 2025-03-02 at 5 46 43 PM" src="https://github.com/user-attachments/assets/8fae022f-8f30-4220-a11c-b26d591d6911" />

Reply:
[The source code comment](https://github.com/suztomo/repo-automation-bots/blob/8624a2d4d429c204b006603d376cb03f6489387b/packages/release-please/src/release-please.ts#L391) says when we create a release, we reload the manifest configuration.